ai_services/anomaly_detection/data_preprocessing_examples/oci_data_flow_based_examples/example_code/pivoting.py

```diff
@@ -32,15 +32,19 @@ def spark_pivoting(df, **kwargs):
         groupby = groupby.split()
     agg = kwargs["agg"]
     agg_ops = dict()
+    distinct_column_values = kwargs['distinct_column_values'] if 'distinct_column_values' \
```

Comment:
```suggestion
    distinct_column_values = kwargs.get('distinct_column_values', None)
```
Please use obtaining default value format instead of `VALUE_1 if EXPRESSION else VALUE_2` format.